.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -296,6 +296,30 @@ jobs:
       - name: Stop cache
         uses: ./.github/actions/post-cache
 
+  test-catch-up:
+    name: Test catching up
+    runs-on: ubuntu-latest
+    needs: build-new-node
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Download all artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: local-tests/
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
```

Comment:
Do you think it would possible to somehow reuse these steps in `test-runtime-update` step to avoid doing the same setup twice?

Reply:
not sure how to do it cleanly but at the same time spent here is negligible compared to other steps in the pipeline.
![Screenshot from 2022-03-23 09-52-45](https://user-images.githubusercontent.com/27210860/159660588-9bb91354-221c-487e-8748-b2355fdba7ee.png)

Reply:
Nevermind then